hardware_interface/include/hardware_interface/tools.hpp

```diff
@@ -0,0 +1,47 @@
+// Copyright 2023 ros2_control Development Team
```

Comment:
Can you name the header `helpers.hpp`?

It's just to maintain consistency with other helpers headers like the one in the controller_interface (https://github.com/ros-controls/ros2_control/blob/master/controller_interface/include/controller_interface/helpers.hpp). What do you think?

Reply:
I don't care actually :grin: